core/src/main/java/net/opentsdb/query/processor/downsample/DownsampleFactory.java

```diff
@@ -138,6 +141,15 @@ public DownsampleConfig parseConfig(final ObjectMapper mapper,
     if (n != null) {
       builder.setTimeZone(n.asText());
     }
+
+    n = node.get("process_as_arrays");
```

Comment:
camel case please.

Reply:
Was following this format from DownsampleConfig.Builder, may be I should not use this format here, will change..